docs/Configuration.md

```diff
@@ -98,10 +98,10 @@ disabled | Can be set to `true` or an arbitrary reason string to disable a user.
 profileContent | Allows to provide [enhanced docview xml](https://jackrabbit.apache.org/filevault/docview.html) that will reset the profile to the given structure after each run | optional
 preferencesContent | Allows to provide [enhanced docview xml](https://jackrabbit.apache.org/filevault/docview.html) that will reset the preferences node to the given structure after each run | optional
 socialContent | Allows to provide [enhanced docview xml](https://jackrabbit.apache.org/filevault/docview.html) that will reset the social node to the given structure after each run | optional
+impersonationAllowedFor | List of users that may impersonate as this user. May be provided as yaml list or comma-separated list (String). If this property is omitted the impersonators of the user remain untouched, otherwise impersonators are added as given in config (and existing ones not configured are removed). To force an empty list of impersonators, an empty yaml list may be used: `impersonationAllowedFor: []`. | optional
```

Comment:
why comma separated list? Just requires additional effort with parsing and would require escaping. Let us stick to plain yaml at least for new features.

Reply:
I'd rather stick with the pattern we have - we can remove comma-support and other yaml quirks we we have in yaml format with version 3.0.0 IMHO

Reply:
Then at least make it clearer that comma-separated lists are the deprecated variant!

Reply:
that is fine, ec644ff98fd6c85